### ISSUE TITLE
To Everyone having the Prop Types issue

### ISSUE THREAD

**Issue by jakallergis** [jaysoo/react-native-prompt#37]
This repo is no longer actively maintained.

Thanks to  [lightman73](https://github.com/lightman73)'s fix some few months ago, we now use [this](https://github.com/jakallergis/rn-prompt) untill @jaysoo gets back on this project.

npm package:  [rn-prompt](https://www.npmjs.com/package/rn-prompt)

**Comment by SkrewEverything** [comment]
I have created a similar project with native UI according to the OS.

Feel free to check it.

npm package: [react-native-input-prompt](https://www.npmjs.com/package/react-native-input-prompt)

**Comment by Jobel91** [comment]
I tried both package, and I think that the second is better than the first, because it is more customisable. 
Thanks to both of you.

**Comment by jakallergis** [comment]
@Jobel91 yeah @SkrewEverything's package is really nice! rn-prompt is only meant as a replacement of @jaysoo's project until he returns to active maintenance.
